setup.py

```diff
@@ -128,8 +134,12 @@ def buildNetworKit(install_prefix, externalCore=False, externalTlx=None, withTes
 	comp_cmd.append("-DNETWORKIT_FLATINSTALL=ON")
 	from sysconfig import get_paths, get_config_var
 	comp_cmd.append("-DNETWORKIT_PYTHON="+get_paths()['include']) #provide python.h files
-	comp_cmd.append("-DNETWORKIT_PYTHON_SOABI="+get_config_var('SOABI')) #provide lib env specification
-	if externalCore:
+	os_soabi = get_config_var('SOABI')
+	if os_soabi is None:
+		os_soabi =  get_config_var('EXT_SUFFIX').split(".")[1] # get_config_var('SOABI') is None on win32-systems
+	comp_cmd.append("-DNETWORKIT_PYTHON_SOABI="+os_soabi) #provide lib env specification
+	if externalCore and sys.platform != "win32":
+		# Since Windows build includes a core dll-lib, builds from different folders are not supported. Hence external core builds are disabled.
```

Comment:
Is that the only reason that externalCore is not supported? AFAICS the real reason is that we cannot build the core as a DLL, right?